README.md

```diff
@@ -46,6 +46,10 @@ The API Reference Guide provides examples of what information is needed for a pa
 ## MetaKey Support
 Meta Key is a key generated by an entity that can be used to authenticate on behalf of other entities provided that the entity which holds key is a parent entity or associated as a partner.
 
+## OAuth Support
+The CyberSource OAuth2.0 Authorization Server (or API Auth Service) will issue access tokens (based on merchant user credentials) to CyberSource or third-party Applications. These applications can access CyberSource API's on the merchant's behalf, using the access tokens.
+During application registration, third-party application developers are issued a client_id and optionally a client_secret (if they can be considered a confidential client, for example a web application). Requesting an access token is fairly straightforward:  Use the CyberSource token URL and include a set of form-urlencoded parameters.
+Upon receipt of this request, the CyberSource Authorization Server validates the user credentials, then mints and returns a response to your application. The contents of this response depend on the type of application. For web server applications, the response is an authorization code which can be exchanged for an access token and (optionally) a refresh token. For client-side applications (e.g. mobile apps), the response is an access token.  [For the Resource Owner Password Flow, only the latter response is returned.]
```

Comment:
Provide some background information for OAuth (or at least provide them a link to the proper documentation). Actual content is here: https://developer.cybersource.com/api/developer-guides/OAuth/cybs_extend_intro.html

Which fields are being used here for OAuth? What do each of these fields mean?

Find a way to provide this information from the table here: [Access Token Request Parameters](https://developer.cybersource.com/api/developer-guides/OAuth/cybs_extend_intro/obtaining_access_refresh_tokens.html)

Line #52 is not required. We do not need to explain what happens for a web server application. The SDK will deal only with client-side applications. Focus on access token generation (from Auth Code and Refresh Token).